packages/clay-color-picker/src/Splotch.tsx

```diff
@@ -33,17 +33,16 @@ const ClayColorPickerSplotch = React.forwardRef<HTMLButtonElement, IProps>(
 		return (
 			<button
 				{...otherProps}
-				className={`btn clay-color-btn ${className ? className : ''}`}
+				className={`${active ? 'active ' : ''}btn clay-color-btn ${
+					requireBorder ? 'clay-color-btn-bordered ' : ''
+				}${className ? className : ''}`}
```

Comment:
To avoid this and organize better, we can add the `classnames` package, as we use it elsewhere, so it would look more or less:

```js
className={classnames('btn clay-color-btn', className, {
	active,
	'clay-color-btn-bordered': requireBorder
})}
```